client/components/ManageProducts/ManageProducts.js

```diff
@@ -59,24 +62,23 @@ class ManageProducts extends React.Component {
               price,
               inventory,
               imageUrl,
-              countryId,
+              country,
             } = product;
-
+            let { categories } = product || [];
+            categories = categories
```

Comment:
is this supposed to be comparative equals?

Reply:
It's just to make sure that categories exists before I use map on it